native/src/modules/app/LocationPermissionManager.js

```diff
@@ -2,15 +2,22 @@
 
 import { check, PERMISSIONS, request, RESULTS } from 'react-native-permissions'
 import { Platform } from 'react-native'
+import buildConfig from './constants/buildConfig'
 
 export const checkLocationPermission = async (): Promise<RESULTS> => {
+  if (buildConfig().featureFlags.fixedCity) {
+    return RESULTS.DISABLED_FOR_FIXED_CITIES
```

Comment:
This option does not exist on `RESULTS`, does it? Even though it works, I think it should be defined. So imo you should create a custom result, e.g.:
```
const PERMISSION_RESULTS = {
  ...RESULTS,
  FORBIDDEN: 'forbidden'
}
```

Reply:
UNAVAILABLE, DENIED and BLOCKED do exist.

Is any of UNAVAILABLE suitable?

Reply:
I think that would work as well :+1:

Reply:
done